architecture/modules/api_gateway/outputs.tf

```diff
@@ -1,7 +1,7 @@
 output "url" {
-    value = google_api_gateway_gateway.gateway.default_hostname
+  value = google_api_gateway_gateway.gateway.default_hostname
 }
 
 output "ip" {
-    value = google_compute_global_address.api_fwd_address.address
+  value = google_compute_global_address.api_fwd_address.address
 }
```

Comment:
Even though terraform says their formatter is unopinionated about vertical whitespace (https://github.com/hashicorp/terraform/issues/23223), it actually removes all of them for me :((((

Reply:
Should I write a cute script to just add them into every file in the architecture folder? It feels like it might be easier/more consistent just to accept their weirdness in this case.

Reply:
It should be available as a setting in your IDE, Jetbrains or VSCode etc, that's probably the easiest route:
- https://stackoverflow.com/a/16761228
- https://stackoverflow.com/a/44704969